service/controller/v24/key/key.go

```diff
@@ -54,7 +54,7 @@ const (
 	CoreosVersion        = "2135.4.0"
 
 	K8SEndpointUpdaterDocker = "quay.io/giantswarm/k8s-endpoint-updater:590479a6228c2c143695a268bda5382b52f7ffe1"
-	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:34e40049f80ce8ea0f774e56c1b10c7f24921eca"
+	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:f3feaa5ba3e8905fc5c1ad9911fc08a0fd1fb66d"
```

Comment:
**XXX**: Update to latest `master` build once tested.

Reply:
So TODO?

Reply:
DONE